README.md

```diff
@@ -44,6 +44,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Cory Knox](#cory-knox) - **streaming:** PowerShell, C#, TypeScript, VSCode extension development
 - [Daniel Jenkins](#daniel-jenkins) - **streaming:** Javascript, PHP, C#, Unity, Game Development 
 - [Daniel Shiffman](#daniel-shiffman) - **streaming:** JavaScript, Processing, P5.js, WebGL, Machine Learning, Algorithms
+- [David Poindexter](#david-poindexter) - **streaming:** Node.js, JavaScript, React, Docker, AWS, Terraform, DevOps, Developer Tooling
```

Comment:
This index internally links to the section below and the anchor link is generated from the header name. Since you included a parenthetical of `(roberttables/mtheoryx)`, it won't match the header anchor.

I checked the rendered README of your branch and the proper anchor is `#david-poindexter-roberttablesmtheoryx`.

Reply:
Hecc, you're right, I didn't even think about that!

Reply:
So I've removed that weird heading, I don't really need it. It should now be more aligned with how others are doing things.